components/test/brave_rewards/extension/brave_rewards/utils_test.ts

```diff
@@ -27,4 +27,25 @@ describe('Rewards Panel extension - Utils', () => {
       expect(convertBalance('10', { 'USD': 10, 'EUR': 4 }, 'EUR')).toBe('40.00')
     })
   })
+
+  describe('handleContributionAmount', () => {
+    it('amount is 0', () => {
+      expect(handleContributionAmount('0')).toBe('0.0')
+    })
+    it('amount is in wrong format', () => {
+      expect(handleContributionAmount('dasdfasdfasdf')).toBe('0.0')
+    })
+    it('amount is probi', () => {
+      expect(handleContributionAmount('1000000000000000000')).toBe('1.0')
+    })
+    it('amount is probi', () => {
+      expect(handleContributionAmount('1540000000000000000')).toBe('1.5')
+    })
+    it('amount is double', () => {
```

Comment:
A test should ideally be added for rounding, i.e. `10.45` which based upon current logic should be `10.4`, but if the rounding is incorrect would be `10.5`

Reply:
done